client/src/ui/components/trading/TransferBetweenEntities.tsx

```
@@ -41,6 +41,126 @@
+    setSelectedStepId: (stepId: STEP_ID) => void;
+  }) => {
+    const isEntitySelected = (entities: any[], selectedEntityId: ID | undefined) => {
+      return entities.some((entity) => entity.entity_id === selectedEntityId);
+    };
+
+    const filterEntities = (entities: any[], searchTerm: string, selectedEntityId: ID | undefined) => {
+      return entities.filter(
+        (entity) =>
+          entity.entity_id === selectedEntityId ||
+          entity.name.toLowerCase().includes(searchTerm.toLowerCase()) ||
+          (entity.accountName && entity.accountName.toLowerCase().includes(searchTerm.toLowerCase())),
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)